cmake/caches/Windows-aarch64.cmake

```diff
@@ -74,7 +74,6 @@ set(LLDB_ENABLE_LIBXML2 NO CACHE BOOL "")
 set(SWIFT_INCLUDE_DOCS NO CACHE BOOL "")
 set(SWIFT_BUILD_ENABLE_PARSER_LIB YES CACHE BOOL "")
 set(SWIFT_BUILD_STDLIB_EXTRA_TOOLCHAIN_CONTENT NO CACHE BOOL "")
-set(SWIFT_BUILD_STDLIB_CXX_MODULE NO CACHE BOOL "")
```

Comment:
Why is this being removed?  We don't build any of the runtime components in the toolchain.

Reply:
Looks like the swiftCxx module is still not built when I specify `SWIFT_BUILD_STDLIB_CXX_MODULE=YES` in the runtime presets.
```
LINK : fatal error LNK1104: cannot open file 'swiftCxx.lib'
```

Reply:
I'm not super familiar with the build pipeline on Windows. Since the Cxx module is shipped with the toolchain, would it be acceptable to build it alongside the toolchain as well?

Reply:
No, that wouldn't.  That would basically force the stdlib to *always* be required, and we cannot do that because it takes too long.